### ISSUE TITLE
Shellcheck scripts directory

### ISSUE THREAD

**Issue by elfenlaid** [SwiftPackageIndex/SwiftPackageIndex-Server#1122]
[Shellcheck](shellcheck) is a linting tool for shell scripts:

<img width="957" alt="Screenshot 2021-05-22 at 15 19 22" src="https://user-images.githubusercontent.com/765790/119226337-34d2cb00-bb11-11eb-9cd7-4824e1eb40bc.png">

So the proposed changes would be:

```diff
#!/bin/sh

set -u

TARFILE=$1

echo "Backing up database to $PWD/$TARFILE ..."

docker run --rm \
-    -v $PWD:/host \
+    -v "$PWD":/host \
    -v spi_db_data:/db_data \
    -w /host \
    ubuntu \
-    tar cfz $TARFILE /db_data
+    tar cfz "$TARFILE" /db_data

echo "done."

# don't let tar errors or warnings bubble up
exit 0
```

Shellcheck has also a CLI. I wonder whether you'll find it helpful. If so, I'll update the scripts.

**Comment by finestructure** [comment]
Ah yes, that's great. I know about shellcheck but I always forget to actually run it 🤣

I'll add those changes or if you want to open up a PR that'll be great, too :+1:

**Comment by elfenlaid** [comment]
> Ah yes, that's great. I know about shellcheck but I always forget to actually run it 🤣
> 
> I'll add those changes or if you want to open up a PR that'll be great, too 👍

Yeah, I would like to open a PR if you don’t mind :)

**Comment by daveverwer** [comment]
Closed by #1135. thanks @elfenlaid!
